### ISSUE TITLE
DNS Problem.. :|

### ISSUE THREAD

**Issue by Qbaakr** [jfroment/seedbox#47]
Hello
Let me start with the fact that this is one of the first projects I have contact with - but where do you need to start...
From the whole package I installed:
-deluge
-traefik
-portainer
-jdownloader
-nextcloud-db
-nextcloud
I did everything according to the settings, log in. and then I encounter an error in the form of no SSL certification for a given DNS address. I have read quite a lot of topics, but due to the fact that this is my beginning - I can't find a solution.
in some topics on http auth there are solutions based on toml, or other solutions, e.g. acme.json file
Thanks in advance with any hint
Regards

P.S.
I tried different options... the option without https works fine

**Comment by jfroment** [comment]
Hello, did you configure your email for Traefik ACME resolver in the .env file (variable is ``ACME_MAIL``)?
Also, is your instance reachable from the Internet on port 80 (maybe check your firewall). By default, Traefik uses the httpChallenge to generate a valid certificate, and this challenges uses the insecure port 80 for Let's Encrypt to check your domain.

If you're still blocked, please do not hesitate to send me your configuration (with password redacted of course), I'll check if I spot something.

**Comment by Qbaakr** [comment]
Hello.
At this point I reinstalled everything. I installed Docker Compose from the official repo, due to the local-presist error I installed it from CWSpear, the rest from the links provided (ubuntu already has jq pre-installed in the newer version). Currently there is this problem: WARN[0000] a network with name traefik-network exists but was not created by compose.
Set `external: true` to use an existing network
network traefik-network was found but has incorrect label com.docker.compose.network set to ""
and now the question is whether it is enough to sign it correctly or delete it and create a new one? 
Regards
